### PR TITLE
Merge release 2 to master

### PR DESCRIPTION
**Pull request by chriselion**
### Proposed change(s)

Merges the release branch back to master.

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: utils/make_readme_table.py [hunk 1]
```diff
@@ -62,6 +62,7 @@ def display_name(self) -> str:
     ReleaseInfo.from_simple_tag("0.15.0", "March 18, 2020"),
     ReleaseInfo.from_simple_tag("0.15.1", "March 30, 2020"),
     ReleaseInfo("release_1", "1.0.0", "0.16.0", "April 30, 2020"),
+    ReleaseInfo("release_2", "1.0.1", "0.16.1", "May 19, 2020"),
```
Comment (chriselion) [inline utils/make_readme_table.py:65]: Will fix this on release_2_branch

Comment (chriselion) [reply utils/make_readme_table.py:65]: never mind, we don't actually use that version in the table. Fixing on this branch instead.

File: docs/Training-ML-Agents.md [hunk 2]
```diff
@@ -206,7 +206,9 @@ behaviors:
     normalize: false
     num_layers: 2
     time_horizon: 64
+    summary_freq: 10000
```
Comment (chriselion) [inline docs/Training-ML-Agents.md:209]: @ervteng Manually applied these from https://github.com/Unity-Technologies/ml-agents/pull/3989/files - can you sanity check me?

**Comment by chriselion** [comment]
Good call, I'll fix those on release_2_branch and update the release_2_docs tag.

**Comment by chriselion** [comment]
Done in https://github.com/Unity-Technologies/ml-agents/pull/4001 and updated https://github.com/Unity-Technologies/ml-agents/tree/release_2_docs

File: docs/Training-ML-Agents.md (hunk 2 above)
Comment (ervteng) [reply docs/Training-ML-Agents.md:209]: 👍

**Pull request merged**